boards/mega-xplained/include/periph_conf.h

```diff
@@ -65,6 +65,22 @@ extern "C" {
 #define TIMER_1_ISRB        TIMER3_COMPB_vect
 /** @} */
 
+/**
+ * @name RTC configuration
+ * @{
+ */
+#define RTC_NUMOF    (1U)
+/** @} */
+
+/**
+ * @name RTT configuration
+ * @{
+ */
+#define RTT_NUMOF        (1U)
+#define RTT_MAX_VALUE    (0x00FFFFFF)    /* 24-bit timer */
+#define RTT_FREQUENCY    (32U)
```

Comment:
This also needs to go for all atmegas. Do you know if all these settings work for all?

Reply:
Added support to jiminy-mega256rfr2. See comment below review.

The settings are based on how the driver works, so they are the same for all users of this driver.